cfgov/unprocessed/css/molecules/big-checkbox.less

```diff
@@ -0,0 +1,108 @@
+.m-big-checkbox {
+    margin-bottom: 15px;
+
+    &:last-child {
+        margin-bottom: 30px;
+    }
+    .a-input {
+        @checkbox-width: unit( 18px / @base-font-size-px, em);
+
+        &[type="checkbox"] {
+            .u-visually-hidden();
+
+            .lt-ie9 & {
+                height: 20px;
+                width: 20px;
+                border: 0;
+                float: left;
+                margin: 1em;
+                position: static;
+                width: auto;
+            }
+        }
+
+        &[type="checkbox"] + label {
+            cursor: pointer;
+            margin-bottom: 0;
+        }
+
+        &[type="checkbox"] + label::before {
+            background: white;
+            content: '\a0';
+            display: inline-block;
+            margin-top: unit( 2px / @base-font-size-px, em);
+            margin-right: .6em;
+        }
+
+        &[type="checkbox"] + label::before {
+            height: @checkbox-width;
+            width: @checkbox-width;
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border;
+        }
+
+        &[type="checkbox"]:checked + label::before {
+            content: @cf-icon-approved;
+            font-family: 'CFPB Minicons';
+            font-size: unit( 16px / @base-font-size-px, em);
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+            text-align: center;
+        }
+
+        &[type="checkbox"]:hover + label::before {
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+        }
+
+        &[type="checkbox"]:focus + label::before {
+            box-shadow: 0 0 0 2px @input-active;
+        }
+
+        &[type="checkbox"]:disabled + label {
+            color: @input-border-disabled;
+        }
+
+        &[type="checkbox"]:disabled + label::before {
+            background: @input-bg-disabled;
+            box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border-disabled;
+            cursor: not-allowed;
+        }
+    }
+
+    label {
+        margin: 0;
+        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                 unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                 unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+    }
+
+    &.selected label,
+    &.focus label,
+    input:focus + label,
+    input:checked + label {
+        outline: 1px solid @input-border-focus;
+    }
+
+    &.selected label,
+    input:checked + label {
+        background-color: @input-inset-selected;
+    }
+
+    &__1 {
```

Comment:
That seems like even more of reason not to add layout code here. If it's just a gray box with the input and label, it's agnostic to any layouts we may want to use now or in the future. The macro probably shouldn't even include the fieldset or layout either, it should just be a single big checkbox.

I'm thinking that it could be organized just like we've done w/ info-units and other molecules. There'd be a container organism that would handle spacing and layout, and that could be the fieldset. That fieldset wouldn't even necessarily have to be related to the big checkbox, it could just be `o-fieldset` and set up layout all on it's own.